package.json

```diff
@@ -49,7 +49,7 @@
   },
   "devDependencies": {
     "@babel/cli": "^7.2.3",
-    "@babel/core": "^7.4.0",
+    "@babel/core": "7.5.5",
```

Comment:
have to set fixed `@babel/core` version to `7.5.5` to avoid using version `7.6.0` where `/* istanbul ignore next */` is ignored.

related issue https://github.com/babel/babel/issues/10432